etc/schema.json

```diff
@@ -1043,7 +1043,8 @@
                                 "additionalProperties": false
                             },
                             "sshfp": {
-                                "description": "A Secure Shell fingerprint, used to verify the system’s authenticity",
+                                "description":
+                                        "A Secure Shell fingerprint, used to verify the system’s authenticity",
```

Comment:
This change does not look related to the commit :-p

Reply:
This and other formatting changes are a result of clang-format.

Reply:
Needs its own commit anyways ;-p